transforms/PathologyLabelSoften.py

```diff
@@ -91,30 +109,50 @@ def _create_soft_labels(self, labels: np.ndarray) -> np.ndarray:
         return labels
 
     def _smooth(self, label: np.ndarray) -> np.ndarray:
-        "Apply mean filtering to 2D numpy array"
+        """Apply median smooth to 2d numpy array"""
```

Comment:
mean filtering is correct. See discussion on monai: https://github.com/Project-MONAI/MONAI/issues/3178